### PR TITLE
Initial template setup for Vite EtageControl project.

### PR DESCRIPTION
**Pull request by Jose-Perigolo**
- Setting EtageControl template, renaming variables and files. Initial setup.

### CODE REVIEW

File: EtageControl/dev/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
+  </head>
+  <body>
+    <div><b>Group Select</b></div>
```
Comment (rjrodger) [inline EtageControl/dev/index.html]: rename to Etage Control

File: EtageControl/vite.config.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+export default defineConfig({
+  build: {
+    minify: false,
+    target: 'es6',
+    emptyOutDir: false,
+    rollupOptions: {
+      treeshake: false,
+      output: {
+        assetFileNames: "group-select.[ext]",
```
Comment (rjrodger) [inline EtageControl/vite.config.js]: rename

File: EtageControl/vite.config.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      },
+    },
+    lib: {
+      entry: resolve(__dirname, 'src/group-select.ts'),
+      name: 'PlantquestGroupSelect',
```
Comment (rjrodger) [inline EtageControl/vite.config.js]: rename

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: EtageControl/dev/index.html (hunk 1 above)
Comment (Jose-Perigolo) [reply EtageControl/dev/index.html]: Done.

File: EtageControl/vite.config.js (hunk 2 above)
Comment (Jose-Perigolo) [reply EtageControl/vite.config.js]: Indeed. Done.

File: EtageControl/vite.config.js (hunk 3 above)
Comment (Jose-Perigolo) [reply EtageControl/vite.config.js]: Indeed. Done.

File: EtageControl/src/etage-control.ts [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    div.classList.add('leaflet-control')
+
+    let ul = L.DomUtil.create('ul')
+    ul.classList.add('leaflet-control-toolbar')
+    ul.classList.add('leaflet-toolbar-0')
+    ul.classList.add('plantquest-tool-building')
+
+    let selectors: HTMLLIElement[] = []
+
+    const buildings = [
```
Comment (rjrodger) [inline EtageControl/src/etage-control.ts:28]: set using an option

**Review by rjrodger — CHANGES_REQUESTED** [review]

**Pull request merged**